infra/chromadb_compaction/lambdas/utils/response.py

```
@@ -0,0 +1,76 @@
+    # If there's an httpMethod, it's from API Gateway
+    if "httpMethod" in event:
+        return False
+
+    # If requestContext exists with apiId, it's from API Gateway
+    if "requestContext" in event and "apiId" in event.get(
+        "requestContext", {}
+    ):
+        return False
+
+    # Otherwise, assume it's from Step Functions or direct invocation
+    return True
```

Ruff (0.12.2)

Return the negated condition directly

(SIM103)